addons/im_livechat/tests/chatbot_common.py

```diff
@@ -40,7 +40,7 @@ def setUpClass(cls):
             cls.step_dispatch_operator,
             cls.step_dispatch_documentation,
         ] = cls.env['chatbot.script.answer'].sudo().create([{
-            'name': 'I want to buy the software',
+            'name': 'I\'d like to buy the software',
```

Comment:
Some motivation for the change? Not that I'm against it, it's actually better worded :)